Plugins/GRpluginStands.txt

```diff
@@ -1003,329 +1048,629 @@ CODE:C
+COORD:N060.12.19.182:E011.04.44.574
+COORD:N060.12.19.349:E011.04.43.382
+COORD:N060.12.20.258:E011.04.43.897
 USE:BP
 CODE:B
 WINGSPAN:20
 PRIORITY:-2
 
 // Military Stands
 //STAND M1
-STAND:ENGM:M1:N060.12.37.184:E011.05.52.867:40
+STAND:ENGM:M1
+COORD:N060.12.37.603:E011.05.54.150
+COORD:N060.12.36.380:E011.05.53.492
+COORD:N060.12.36.887:E011.05.49.793
+COORD:N060.12.38.112:E011.05.50.498
+COORD:N060.12.37.603:E011.05.54.150
 ATYP:C130
 USE:T
 WINGSPAN:41
 
 //STAND M2
-STAND:ENGM:M2:N060.12.35.512:E011.05.51.965:40
+STAND:ENGM:M2
+COORD:N060.12.36.077:E011.05.53.273
+COORD:N060.12.34.854:E011.05.52.615
+COORD:N060.12.35.361:E011.05.48.916
+COORD:N060.12.36.586:E011.05.49.621
+COORD:N060.12.36.077:E011.05.53.273
 ATYP:C130
 USE:T
 WINGSPAN:41
 
 //STAND M3
-STAND:ENGM:M3:N060.12.33.893:E011.05.51.214:40
+STAND:ENGM:M3
+COORD:N060.12.34.526:E011.05.52.361
+COORD:N060.12.33.303:E011.05.51.704
+COORD:N060.12.33.810:E011.05.48.004
+COORD:N060.12.35.035:E011.05.48.709
+COORD:N060.12.34.526:E011.05.52.361
 ATYP:C130
 USE:T
 WINGSPAN:41
 
 //STAND M5
-STAND:ENGM:M5:N060.12.28.935:E011.05.48.136:20
+STAND:ENGM:M5
+COORD:N060.12.30.128:E011.05.47.412
+COORD:N060.12.29.778:E011.05.49.873
+COORD:N060.12.27.784:E011.05.48.769
+COORD:N060.12.28.147:E011.05.46.249
+COORD:N060.12.30.128:E011.05.47.412
 USE:IMT
 
 //STAND M6
-STAND:ENGM:M6:N060.12.28.516:E011.05.51.085:20
+STAND:ENGM:M6
+COORD:N060.12.29.717:E011.05.50.363
+COORD:N060.12.29.366:E011.05.52.824
+COORD:N060.12.27.372:E011.05.51.719
+COORD:N060.12.27.735:E011.05.49.200
+COORD:N060.12.29.717:E011.05.50.363
 USE:IMT
 
 //STAND M7
-STAND:ENGM:M7:N060.12.28.121:E011.05.53.898:20
+STAND:ENGM:M7
+COORD:N060.12.29.298:E011.05.53.229
+COORD:N060.12.28.947:E011.05.55.690
+COORD:N060.12.26.953:E011.05.54.585
+COORD:N060.12.27.316:E011.05.52.066
+COORD:N060.12.29.298:E011.05.53.229
 USE:IMT
 
 //STAND M8
-STAND:ENGM:M8:N060.12.27.675:E011.05.56.837:20
+STAND:ENGM:M8
+COORD:N060.12.28.876:E011.05.56.143
+COORD:N060.12.28.525:E011.05.58.604
+COORD:N060.12.26.531:E011.05.57.499
+COORD:N060.12.26.894:E011.05.54.980
+COORD:N060.12.28.876:E011.05.56.143
 USE:IMT
 
 //STAND M9
-STAND:ENGM:M9:N060.12.27.296:E011.05.59.782:20
+STAND:ENGM:M9
+COORD:N060.12.28.464:E011.05.58.933
+COORD:N060.12.28.113:E011.06.01.394
+COORD:N060.12.26.119:E011.06.00.289
+COORD:N060.12.26.482:E011.05.57.770
+COORD:N060.12.28.464:E011.05.58.933
 USE:IMT
 
 //STAND M10
-STAND:ENGM:M10:N060.12.26.897:E011.06.02.702:20
+STAND:ENGM:M10
+COORD:N060.12.28.037:E011.06.01.856
+COORD:N060.12.27.687:E011.06.04.318
+COORD:N060.12.25.693:E011.06.03.213
+COORD:N060.12.26.056:E011.06.00.693
+COORD:N060.12.28.037:E011.06.01.856
 USE:IMT
 
 //STAND M20
-STAND:ENGM:M20:N060.13.10.030:E011.06.12.205:20
+STAND:ENGM:M20
+COORD:N060.13.10.963:E011.06.12.343
+COORD:N060.13.10.278:E011.06.14.135
+COORD:N060.13.08.843:E011.06.11.886
+COORD:N060.13.09.540:E011.06.10.148
+COORD:N060.13.10.963:E011.06.12.343
 USE:M
 PRIORITY:-3
```

Comment:
Would remove autoassignment here - would be very weird to park someone here based on automatic stand assignment.

Reply:
Changed to manual in 8fb4d468111bf0578cbe7a0d88e24724d666829b.